packages/lesswrong/components/sunshineDashboard/SunshineNewUsersItem.tsx

```diff
@@ -215,29 +222,39 @@ const SunshineNewUsersItem = ({ user, classes, updateUser, allowContentPreview=t
               <div dangerouslySetInnerHTML={{__html: user.htmlBio}}/>
               <hr className={classes.hr}/>
               <div className={classes.row}>
-                {!!(user.maxCommentCount || user.maxPostCount) && <LWTooltip title="Review">
-                  <Button onClick={handleReview}>
-                    <DoneIcon /> Review
-                  </Button>
-                </LWTooltip>}
-                <LWTooltip title="Snooze (approve all posts)">
-                  <Button title="Snooze" onClick={handleSnooze}>
-                    <SnoozeIcon /> Snooze
-                  </Button>
-                </LWTooltip>
-                {!user.reviewedByUserId && <LWTooltip title="Purge (delete and ban)">
-                  <Button onClick={handlePurge}>
-                    <DeleteForeverIcon /> Purge
-                  </Button>
-                 </LWTooltip>}
-                {user.reviewedByUserId && <LWTooltip title="">
-                  <Button onClick={handleBan}>
-                    <RemoveCircleOutlineIcon /> Ban
-                  </Button>
-                </LWTooltip>}
-                {currentUser && <NewConversationButton user={user} currentUser={currentUser} templateCommentId={templateId}>
-                  <Button variant="outlined">Message</Button>
-                </NewConversationButton>}
+                <div className={classes.row}>
+                  {!!(user.maxCommentCount || user.maxPostCount) && <LWTooltip title="Approve">
+                    <Button onClick={handleReview}>
+                      <DoneIcon />
+                    </Button>
+                  </LWTooltip>}
+                  <LWTooltip title="Snooze (approve all posts)">
+                    <Button title="Snooze" onClick={handleSnooze}>
+                      <SnoozeIcon />
+                    </Button>
+                  </LWTooltip>
+                  <LWTooltip title="Ban for 3 months">
+                    <Button onClick={handleBan}>
+                      <RemoveCircleOutlineIcon />
+                    </Button>
+                  </LWTooltip>
+                  {!user.reviewedByUserId && <LWTooltip title="Purge (delete and ban)">
+                    <Button onClick={handlePurge}>
+                      <DeleteForeverIcon />
+                    </Button>
+                  </LWTooltip>}
+                </div>
+                <div className={classes.row}>
+                  {currentUser && <Select value={0} variant="outlined">
+                    <MenuItem value={0}>Start a message</MenuItem>
+                    {defaultModeratorComments.get().map((template, i) => <MenuItem value={10}>
```

Comment:
What does it mean for a MenuItem to have a `value`?

Reply:
It's how the Select-choice is determined. That said, it's a bit weird, since I'm using the select as a dropdown menu, and actually upon reflection I think it's not even the right choice here, I should basically just be using Menu instead of Select, and not sure why I didn't think of that either.

Reply:
Oh, also it doesn't make any goddamn sense for the value to always be 10, I think I just finished this late last night and didn't do a sanity-check-pass.